### ISSUE TITLE
`sgx_ql_qve_collateral_t::tee_type` is missing in the documentation

### ISSUE THREAD

**Issue by KB5201314** [intel/SGXDataCenterAttestationPrimitives#277]
In [code of version 1.15](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/dcap_1.15_reproducible/QuoteGeneration/quote_wrapper/common/inc/sgx_ql_lib_common.h#LL210C14-L210C14)

```c
typedef struct _sgx_ql_qve_collateral_t
{
    union {
        uint32_t version;           ///< 'version' is the backward compatible legacy representation
        struct {                    ///< For PCS V1 and V2 APIs, the major_version = 1 and minor_version = 0 and
            uint16_t major_version; ///< the CRLs will be formatted in PEM. For PCS V3 APIs, the major_version = 3 and the
            uint16_t minor_version; ///< minor_version can be either 0 or 1. minor_verion of 0 indicates the CRL’s are formatted
                                    ///< in Base16 encoded DER.  A minor version of 1 indicates the CRL’s are formatted in raw binary DER.
        };
    };
    uint32_t tee_type;                     ///<  0x00000000: SGX or 0x00000081: TDX
// ...
// ...
} sgx_ql_qve_collateral_t;
```

There is a field named `tee_type`.

However, in the DCAP ECDSA Quote Library API [documentation](https://download.01.org/intel-sgx/sgx-dcap/1.15/linux/docs/Intel_SGX_ECDSA_QuoteLibReference_DCAP_API.pdf)  version 1.15, page 57, there is no such field in the `sgx_ql_qve_collateral_t` structure. Is this a typo of the code or the documentation?

Please correct me if there is any problem in my understanding, thanks :)

**Comment by llly** [comment]
Thanks. The change is missed in the document. Will update.

**Comment by llly** [comment]
Fixed in document for DCAP 1.16 release.
